src/screens/SettingsScreen.tsx

```diff
@@ -5,27 +5,26 @@ import { MaterialIcons } from '@expo/vector-icons';
 import Colors from '../stylesheets/colors';
 import { getAccount, logout } from '../services/auth';
 import type { MSALAccount } from 'react-native-msal';
-import { useNavigation } from '@react-navigation/native';
 
 const SettingsScreen = (props: {
   config: Array<{ icon: string; title: string; route: string }>;
   onLogout?: () => void;
   routeAfterLogout: string;
   backLabel?: string;
+  navigation: any;
```

Comment:
Preben is too quick for me. Adding a comment nonetheless. Could the navigation type be better?

Reply:
@dbjoEq